src/store/reducers/appReducer.js

```diff
@@ -200,7 +243,9 @@ const fetchableTypes = [
   epics.actionTypes.fetchDeviceGroupFilters,
   epics.actionTypes.fetchAzureMapsKey,
   epics.actionTypes.updateLogo,
-  epics.actionTypes.fetchLogo
+  epics.actionTypes.fetchLogo,
+  epics.actions.getSolutionSettings,
+  epics.actions.updateSolutionSettings
```

Comment:
"Update" is not a fetch... so it shouldn't be listed in the fetchable types.

Reply:
fixed